lib/engine/round/route_bonus.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
```

Comment:
not sure we should have this class

Reply:
I will try to use hex_bonus (which I have renamed to hexes_bonus) for both this cases.